### PR TITLE
Feature/update db schema

### PR DESCRIPTION
**Pull request by Sontranwakumo**
### Description
This PR include works:
- Create entities in `src/entities` folder follow [database design](https://drive.google.com/file/d/1DbxZnAVpJCKPpsHVBtfO9lKhnKZnd4Vf/view?usp=sharing)
- Create migration to initialize database in `src/migration` and database module `src/database/database.module.ts`
- Deleted unused file `app.controller.ts` `app.services.ts`
- Config in `typeorm.config.ts`,`package.json`.
- Migrate my local database.

### Details:
Created entities:
- user
- branch
- courtInfo
- orderform
- payment, payment_detail
- timeslot
- open_schedule, off_schedule, default_price
- comment

Migrations:
- `src/migration/1721965225854-InitializeDB.ts`

### Check tutorial 
- Run command `npm run db:migrate:up` to apply migration. See detail database configuration in `typeorm.config.ts`.
- Check if tables in database are qualified

### CODE REVIEW

File: src/entities/courtinfo.entity.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+    @ManyToOne(()=>Branch, branch=> branch.courts)
+    @JoinColumn({name:'id_branch'})
+    branch: Branch
+
+    @Column({default:true})
+    is_active: boolean;
+
+    @Column({default:false})
+    is_locked: boolean;
```
Comment (longhoangwkm) [inline src/entities/courtinfo.entity.ts]: is_active, is_locked are also status, we can use `status` enum instead, status: [active, :deactivate, :lock]

File: src/entities/default_price.entity.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+@Entity()
+export class DefaultPrice {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @ManyToOne(() => Branch, branch => branch.defaultPrices)
+  @JoinColumn({ name: 'branch_id' })
+  branch: Branch;
+
+  @Column('time')
```
Comment (longhoangwkm) [inline src/entities/default_price.entity.ts:13]: nullable:true

File: src/entities/open_schedule.entity.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+export class OpenSchedule {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @ManyToOne(() => Branch, branch => branch.openSchedules)
+  @JoinColumn({ name: 'branch_id' })
+  branch: Branch;
+
+  @Column('time')
+  start_time: string;
```
Comment (longhoangwkm) [inline src/entities/open_schedule.entity.ts:20]: nullable:true, allow some branch can open 24/24

File: src/entities/courtinfo.entity.ts (hunk 1 above)
Comment (Sontranwakumo) [reply src/entities/courtinfo.entity.ts]: Thank you, I will update and create new migration.

File: src/entities/open_schedule.entity.ts (hunk 3 above)
Comment (Sontranwakumo) [reply src/entities/open_schedule.entity.ts:20]: Thank you, I will update it

File: src/migration/1721979494668-ChorePropertiesChange.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class ChorePropertiesChange1721979494668 implements MigrationInterface {
```
Comment (longhoangwkm) [inline src/migration/1721979494668-ChorePropertiesChange.ts]: AddStatusToCourInfos

**Pull request merged**